src/pages/Main/Manual/Manual.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
I don't find use "React" in this component. Is he needed? And another place too

Reply:
fix this